api/lightwallet_rest.md

```diff
@@ -309,11 +318,12 @@ was actually spent.
 
 **Response** object
 
-|    Field   |        Type       |                Description              |
-|------------|-------------------|-----------------------------------------|
-| per_kb_fee | uint64            | Estimated network fee                   |
-| amount     | uint64-string     | The total value in outputs              |
-| outputs    | array of output's | Outputs possibly available for spending |
+|    Field     |        Type       |                Description              |
+|--------------|-------------------|-----------------------------------------|
+| per_byte_fee | uint64-string     | Estimated network fee                   |
+| fee_mask     | uint64-string     | Fee quantization mask                   |
```

Comment:
This can _probably_ be a `uint64` field, because it is unlikely that this mask will exceed 2^53. This would be masking 1000 XMR or so.